oauth2/main.go

```diff
@@ -347,3 +302,42 @@ func generateString() (string, error) {
 	}
 	return string(b), nil
 }
+
+func promptForPAT(integration string) error {
+	metadata := map[string]string{
+		"authType":        "pat",
+		"toolContext":     "credential",
+		"toolDisplayName": fmt.Sprintf("%s%s Integration", strings.ToTitle(integration[:1]), integration[1:]),
+	}
+
+	b, err := json.Marshal(metadata)
+	if err != nil {
+		return fmt.Errorf("promptForPAT: failed to marshal metadata: %w", err)
+	}
+
+	g, err := gptscript.NewGPTScript(gptscript.GlobalOptions{})
+	if err != nil {
+		return fmt.Errorf("promptForPAT: failed to create GPTScript client: %w", err)
+	}
+	defer g.Close()
+
+	run, err := g.Run(context.Background(), "sys.prompt", gptscript.Options{
+		Input: fmt.Sprintf(`{"metadata": %s,"message":"Please enter your personal access token for %s.","fields":"Token","sensative": true}`, b, integration),
```

Comment:
```suggestion
		Input: fmt.Sprintf(`{"metadata": %s,"message":"Please enter your personal access token for %s.","fields":"Token","sensitive": true}`, b, integration),
```

If this typo exists on the gptscript end, then it should be fixed there too.